Gemfile.lock

```diff
@@ -8,21 +8,21 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activemodel (7.0.4)
-      activesupport (= 7.0.4)
-    activerecord (7.0.4)
-      activemodel (= 7.0.4)
-      activesupport (= 7.0.4)
-    activesupport (7.0.4)
+    activemodel (7.0.4.3)
+      activesupport (= 7.0.4.3)
+    activerecord (7.0.4.3)
+      activemodel (= 7.0.4.3)
+      activesupport (= 7.0.4.3)
+    activesupport (7.0.4.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    concurrent-ruby (1.1.10)
+    concurrent-ruby (1.2.2)
     diff-lcs (1.5.0)
-    i18n (1.12.0)
+    i18n (1.13.0)
       concurrent-ruby (~> 1.0)
-    minitest (5.17.0)
+    minitest (5.18.0)
     pg (1.4.5)
```

Comment:
Probably not directly related to vulnerabilities but we might want to upgrade the `pg` gem as well. There have been many changes and improvements since the current version.

Improvements: https://github.com/ged/ruby-pg/blob/master/History.md

Latest version: https://github.com/ged/ruby-pg/releases/tag/v1.5.3

Reply:
Yes, you are right that this this upgrade is not linked to vulnerabilities. As this is third party gem I don't want to make major changes that might have impact but also I do have a plan to first upgrade the version of pg in our service. https://github.com/accredible/accredible-credential-api/ for which the PR will be created soon. Im just testing it.

Reply:
Got it! That makes sense. It might be worth noting that you might end up redoing some of the work here if pg has dependencies on some of the gems being currently updated. 

But it might be safer to do it in parts even if the two repositories seem independent.